docs/design/coreclr/botr/clr-abi.md

```diff
@@ -79,6 +81,8 @@ However, unlike native varargs, all floating point arguments are not promoted to
 
 Managed varargs are not supported in .NET Core.
```

Comment:
```suggestion
Managed varargs are not supported on Windows only.
```
The doc was not updated when managed varargs support was added on Windows in .NET Core 3.

Reply:
Correct: *are supported*
Also clarify: is it supported on Windows ARM64?